src/scarab.css

```diff
@@ -150,6 +179,39 @@
   margin-left: -1px;
 }
 
+/* adding info logo and info link */
+
+.mz-bug-info{
+  width: 48px;
+  height: 48px;
+
+  position: absolute;
+  left: 288px;
+ 
+  border-left: 1px solid rgba(0,0,0,0.10);
+}
+
+.mz-bug-info-logo {
+  background-image: url(http://png-1.vector.me/files/images/3/9/394040/universal_information_symbol_thumb);
```

Comment:
I think we need to do a little bit of extra homework on this image. We need to make sure we have the rights to use and distribute it -- or we might create our own. Ask @souperneon for help on this.

Once we have the correct asset, we need to host it in this repository. (3rd party websites can go down or move files without our knowledge, so we can't depend on using those resources.) We can either put the file in this directory (https://github.com/mapzen/scarab/tree/master/src/images) where the Mapzen logo already is, or embed it as a data-URL like other images here.